### ISSUE TITLE
An error("OALSimpleAudio.h") is generated when user build and run the Magical.spriteBuilder project on Xcode. 

### ISSUE THREAD

**Issue by Shahidwerplay** [spritebuilder/SpriteBuilder#1427]
1.Clone Physics Demo from https://github.com/vlidholt/Magician.spritebuilder
2.Launch Application.
3.Open the project Magical.spritebuilder
4.Publish the project on SB.
5. Build and Run the project on XCode.

--Result: An error("OALSimpleAudio.h") is generated when user run the Magical.spriteBuilder project on Xcode. 

Device:
Mac OS: 10.10.0
Xcode: 6.1
SB Version: 1.4.7
SBAndroid version: SBAndroidPluginIndie_58792478 and SBAndroidPluginStarter_58792478

Project Link: https://drive.google.com/file/d/0B5-RfBatv-b8d2ZHTXZlcHRHLVE/view?usp=sharing

![magicianerror](https://cloud.githubusercontent.com/assets/11208448/6616272/2c9f0d54-c8ce-11e4-966b-51e979519009.png)

**Comment by cocojoe** [comment]
Did you allow SB to update the project?

**Comment by Shahidwerplay** [comment]
Yes I did. Plus I removed the project and reinstalled it again but the issue still prevails.

**Comment by sana20** [comment]
This issue is re-occurring.

Mac OS X: 10.10.1
XCode: 6.1.
SBAndroid Version: SBAndroid Indie 948d0c4e
Spritebuilder Version: 1.4.9.

**Comment by Shahidwerplay** [comment]
This issue is re-occurring.

Mac OS X: 10.10.0
XCode: 6.1.
SBAndroid Version: SBAndroidPluginIndie-sbandroid-indie-578c109f92ba6615.1
Spritebuilder Version: 1.4.9.

**Comment by Shahidwerplay** [comment]
This issue is re-occurring.

Mac OS X: 10.10.0
XCode: 6.1.
SBAndroid Version: SBAndroidPluginProfessional-sbandroid-professional-092e93d904850685
Spritebuilder Version: 1.4.9.
